python/lsst/rubintv/production/rubinTv.py

```diff
@@ -825,3 +850,201 @@ def beat(self, ensure=False, customFlatlinePeriod=None):
+    upload to the table.
+    """
+    def __init__(self, outputRoot, *, doRaise=False, embargo=False):
+        self.dataProduct = 'quickLookExp'
+        self.uploader = Uploader()
+        self.butler = makeDefaultLatissButler(embargo=embargo, writeable=True)
+        self.log = _LOG.getChild("monitorChannel")
+        self.channel = 'auxtel_characterizeImage'
+        self.watcher = Watcher(self.dataProduct, self.channel)
+        self.fig = plt.figure(figsize=(12, 12))
+        self.doRaise = doRaise
+        self.shardsDir = os.path.join(outputRoot, 'shards')
+        self.outputRunName = "LATISS/runs/quickLook/1"
+
+        config = CharacterizeImageConfig()
+        basicConfig = CharacterizeImageConfig()
+        obs_lsst = getPackageDir("obs_lsst")
+        config.load(os.path.join(obs_lsst, "config", "characterizeImage.py"))
+        config.load(os.path.join(obs_lsst, "config", "latiss", "characterizeImage.py"))
+        config.measurement = basicConfig.measurement
+
+        config.doApCorr = False
+        config.doDeblend = False
+        self.charImage = CharacterizeImageTask(config=config)
+
+        config = CalibrateConfig()
+        basicConfig = CalibrateConfig()
+        config.load(os.path.join(obs_lsst, "config", "calibrate.py"))
+        config.load(os.path.join(obs_lsst, "config", "latiss", "calibrate.py"))
+        config.measurement = basicConfig.measurement
+
+        config.doApCorr = False
+        config.doDeblend = False
+
+        self.calibrate = CalibrateTask(config=config, icSourceSchema=self.charImage.schema)
+
+    def _getRefObjLoader(self, refcatName, dataId, config):
+        """Get a reference object loader for a given refcat and dataId.
+        """
+        refs = self.butler.registry.queryDatasets(refcatName, dataId=dataId).expanded()
+        refs = list(refs)
+        handles = [dafButler.DeferredDatasetHandle(butler=self.butler, ref=ref, parameters=None)
+                   for ref in refs]
+        dataIds = [ref.dataId for ref in refs]
+
+        loader = ReferenceObjectLoader(
+            dataIds,
+            handles,
+            name=refcatName,
+            log=self.log,
+            config=config
+        )
+        return loader
+
+    def callback(self, dataId, **kwargs):
+        """Method called on each new dataId as it is found in the repo.
+
+        Runs on the quickLookExp and writes shards with various measured
+        quantities, as calculated by the CharacterizeImageTask and
+        CalibrateTask.
+        """
+        try:
+            tStart = time.time()
+
+            self.log.info(f'Running Image Characterization for {dataId}')
+            exp = _waitForDataProduct(self.butler, self.dataProduct, dataId, self.log)
+
+            if not exp:
+                raise RuntimeError(f'Failed to get {self.dataProduct} for {dataId}')
+
+            if isDispersedExp(exp):
+                self.log.info(f'Skipping dispersed image: {dataId}')
+                return
```

Comment:
Let's also file a ticket for both turning this off to see how well it can manage dispersed images, and also stopping it flailing on calibs. These can all be filed on me, of course.